packages/frontend/src/scripts/index.ts

```diff
@@ -1,2 +1,18 @@
+import { formatLargeNumber } from '../pages/formatLargeNumber'
+import { tvlElId } from '../pages/home/Home'
```

Comment:
Can you make sure that this is tree shaken so that only the id exists in the final code?

Reply:
Done